java/mapreduce/src/test/org/apache/orc/mapreduce/TestMapreduceOrcOutputFormat.java

```diff
@@ -46,8 +46,10 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.net.URL;
```

Comment:
This seems to be unused.

BTW, this means that our GitHub Action CI lose Checkstyle test coverage.

Reply:
Good catch! Just removed the unused import.
Yes we are probably missing something here -- any chance checkstyle is ignored for Tests?

Reply:
I'll take a look at it~